### PR TITLE
Add note about ESX and ESXi

### PR DESCRIPTION
**Pull request by cbudz**
Hey Dayle,

*new PR here*

I've added a note to this file per the request laid out in BZ1364569.

Please find a preview of the document here:

http://file.rdu.redhat.com/~cbudzilo/CloudForms/smartstate/build/tmp/en-US/html-single/#performing_smartstate_analysis_on_hosts

-Chris

### CODE REVIEW

File: doc-Managing_Infrastructure_and_Inventory/topics/Performing_SmartState_Analysis_on_Hosts.adoc [hunk 1]
```diff
@@ -1,7 +1,10 @@
 = Performing SmartState Analysis on Hosts
 
 Perform a SmartState Analysis on a host to collect additional information about it, such as patches, CPU, and memory.
```
Comment (suyogsainkar) [inline doc-Managing_Infrastructure_and_Inventory/topics/Performing_SmartState_Analysis_on_Hosts.adoc]: Analysis > analysis

File: doc-Managing_Infrastructure_and_Inventory/topics/Performing_SmartState_Analysis_on_Hosts.adoc [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
-
+[NOTE]
+====
+When VMware ESX or ESXi hypervisors are present please consider the following: *ESX* hosts utilize a service console for host management and can be accessed using SSH. *ESXi* hosts lack a service console and therefore SSH cannot be used to obtain information sets for patches, services, Linux packages, user groups, SSH Config, and FS Files. 
+====
 [NOTE]
 ====
```
Comment (suyogsainkar) [inline doc-Managing_Infrastructure_and_Inventory/topics/Performing_SmartState_Analysis_on_Hosts.adoc]: Instead of back-to-back note boxes I'd combine them into one containing two bullet points.

File: doc-Managing_Infrastructure_and_Inventory/topics/Performing_SmartState_Analysis_on_Hosts.adoc [hunk 3]
```diff
@@ -1,7 +1,10 @@
 = Performing SmartState Analysis on Hosts
 
 Perform a SmartState Analysis on a host to collect additional information about it, such as patches, CPU, and memory.
-
+[NOTE]
+====
+When VMware ESX or ESXi hypervisors are present please consider the following: *ESX* hosts utilize a service console for host management and can be accessed using SSH. *ESXi* hosts lack a service console and therefore SSH cannot be used to obtain information sets for patches, services, Linux packages, user groups, SSH Config, and FS Files.
```
Comment (suyogsainkar) [inline doc-Managing_Infrastructure_and_Inventory/topics/Performing_SmartState_Analysis_on_Hosts.adoc]: Could you rephrase the first line to make it simpler? Perhaps, "For ESX or ESXi hypervisors, consider the following:" Also I'd remove the word ~~please~~.

**Comment by cbudz** [comment]
Hey @suyogsainkar - this PR was affected when I scrapped my repo, so I've had to open a new PR based on your feedback, which you'll find here: PR[405](https://github.com/ManageIQ/manageiq_docs/pull/405).

I'll close this one once the other is complete.

**Comment by suyogsainkar** [comment]
No worries, @cbudz. Thanks for the changes via https://github.com/ManageIQ/manageiq_docs/pull/405. I've merged it to master. You can close this one out now. Cheers, Suyog

**Comment by cbudz** [comment]
Closing
